pyproject.toml

```
@@ -32,7 +32,7 @@ classifiers = [
 ]
 requires-python = ">=3.7"
 dependencies = [
-    "xmltodict==0.12",
+    "xmltodict<0.14",
```

This seems more appropriate to me: it allows 0.13, and requires at least 0.12, so someone wouldn't accidentally install a lower version than it was previously possible.

```suggestion
    "xmltodict>=0.12,<0.14",
```